### ISSUE TITLE
npm install node-sass FAILED (Mac OS X 10.6.8, node v4.2.1, npm  v2.14.7 )

### ISSUE THREAD

**Issue by ado-san** [sass/node-sass#1240]
npm install node-sass Failed at the node-sass@3.4.1 postinstall script
_________________________________________
Environment 
Mac OS X 10.6.8
Xcode 3.2.6
node v4.2.1
npm  v2.14.7
__________________________________________
Steps
running npm install node-sass
expected node-sass installed succesfully
result installation failed code ELIFECYCLE
target install ionic via npm install -g ionic
tried npm cache clean, npm install --unsafe-perm -g node-sass, npm rebuild node-sass
__________________________________________
npm-debug.log extract
------------------------------
16084 verbose pkgid node-sass@3.4.1
16085 verbose cwd /Users/buyer/ionictodo
16086 error Darwin 10.8.0
16087 error argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "node-sass"
16088 error node v4.2.1
16089 error npm  v2.14.7
16090 error code ELIFECYCLE
16091 error node-sass@3.4.1 postinstall: `node scripts/build.js`
16091 error Exit status 1
16092 error Failed at the node-sass@3.4.1 postinstall script 'node scripts/build.js'.
16092 error This is most likely a problem with the node-sass package,
16092 error not with npm itself.
16092 error Tell the author that this fails on your system:
16092 error     node scripts/build.js
16092 error You can get their info via:
16092 error     npm owner ls node-sass
16092 error There is likely additional logging output above.
16093 verbose exit [ 1, true ]
16094 verbose unbuild node_modules/node-sass
16095 info preuninstall node-sass@3.4.1
16096 info uninstall node-sass@3.4.1
16097 verbose unbuild rmStuff node-sass@3.4.1 from /Users/buyer/ionictodo/node_modules
16098 silly gentlyRm /Users/buyer/ionictodo/node_modules/.bin/node-sass is being gently removed
16099 silly gentlyRm verifying /Users/buyer/ionictodo is an npm working directory
______________________________________________________________________
nmp.log extract
--------------------
spawn-sync@1.0.14 postinstall /Users/buyer/ionictodo/node_modules/node-sass/node_modules/cross-spawn/node_modules/spawn-sync
node postinstall

node-sass@3.4.1 install /Users/buyer/ionictodo/node_modules/node-sass
node scripts/install.js

Binary downloaded and installed at /Users/buyer/ionictodo/node_modules/node-sass/vendor/darwin-x64-46/binding.node

node-sass@3.4.1 postinstall /Users/buyer/ionictodo/node_modules/node-sass
node scripts/build.js

` /Users/buyer/ionictodo/node_modules/node-sass/vendor/darwin-x64-46/binding.node ` exists. 
 testing binary.
Problem with the binary.
Manual build incoming.
Building: /usr/local/bin/node /Users/buyer/ionictodo/node_modules/node-sass/node_modules/node-gyp/bin/node-gyp.js rebuild --verbose --libsass_ext= --libsass_cflags= --libsass_ldflags= --libsass_library=
  g++ '-DNODE_GYP_MODULE_NAME=libsass' '-D_DARWIN_USE_64_BIT_INODE=1' '-D_LARGEFILE_SOURCE' '-D_FILE_OFFSET_BITS=64' '-DLIBSASS_VERSION="3.3.1"' -I/Users/buyer/.node-gyp/4.2.1/include/node -I/Users/buyer/.node-gyp/4.2.1/src -I/Users/buyer/.node-gyp/4.2.1/deps/uv/include -I/Users/buyer/.node-gyp/4.2.1/deps/v8/include -I../src/libsass/include  -Os -gdwarf-2 -mmacosx-version-min=10.7 -arch x86_64 -Wall -Wendif-labels -W -Wno-unused-parameter -std=c++11 -stdlib=libc++ -fno-threadsafe-statics -fno-strict-aliasing -MMD -MF ./Release/.deps/Release/obj.target/libsass/src/libsass/src/ast.o.d.raw  -c -o Release/obj.target/libsass/src/libsass/src/ast.o ../src/libsass/src/ast.cpp
______________________________________________________________

**Comment by xzyfer** [comment]
Thanks @ado-san. Please follow the [troubleshooting guide](https://github.com/sass/node-sass/blob/master/TROUBLESHOOTING.md) as previously asked to.

Log extract are typically not useful. Please provide the **entire** output from the troubleshooting steps in a github gist. Please **do not** post the output here.

**Comment by DabsDesign** [comment]
I'm having the same issue here, but I stumbled upon this problem when trying to install the gulp-sass package. This file cannot be found by the npm installer:
```
https://github.com/sass/node-sass/releases/download/v3.4.1/win32-x64-47_binding.node
```
If you go to the releases directory you can see that the ```win32-x64-47_binding.node``` file is not present, only it's earlier version.

**Comment by xzyfer** [comment]
Windows Node 5 binary is now available for v3.4.1 **only**. Just npm install.

**Comment by benclarke** [comment]
sorry, to clarify, how do I resolve this problem? do <code>npm install</code> instead of <code>npm install node-sass</code>?
